quickstarts/aws/aws-network-firewall/config.yml

```diff
@@ -0,0 +1,31 @@
+slug: aws-network-firewall
+description: |-
+  ## What is AWS Network Firewall?
+  AWS Network Firewall is a stateful, managed, network firewall and intrusion detection and prevention service for your virtual private cloud (VPC) that you create in Amazon Virtual Private Cloud (Amazon VPC). With Network Firewall, you can filter traffic at the perimeter of your VPC.
+  ### Get started!
+  Start monitoring AWS Network Firewall by connecting Amazon Web Services (AWS) to New Relic! Take a look at our documentation for information on how to set up this integration.
+  ### More info
+  Check out the [documentation](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-network-firewall-monitoring-integration/) to learn more about New Relic monitoring for AWS Network Firewall.
+summary: |-
+  Monitor AWS Network Firewall by connecting AWS to New Relic
+icon: logo.png
+level: New Relic
+authors:
+  - New Relic
+title: AWS Network Firewall
+documentation:
+  - name: AWS Network Firewall installation docs
+    description: |
+      Monitor AWS Network Firewall by connecting AWS to New Relic.
+    url: >-
+      https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-network-firewall-monitoring-integration/
```

Comment:
It looks like this is a 404? Have the docs not released yet?

Reply:
Hey @d3caf,
I've submitted a Pull Request for the documentation, which you can find here: [PR #16235](https://github.com/newrelic/docs-website/pull/16235). It should be merged shortly.

Reply:
@d3caf , Documentation is live now.